src/lib/share/__tests__/generateReceiveShareObject.js

```diff
@@ -5,7 +5,7 @@ describe('generateReceiveShareObject', () => {
     // Given
     const title = 'Sending G$ via GoodDollar App'
     const text = "Joe Bloggs, You've got a request from John Doe for 1 G$. To Transfer open:"
-    const isReceiveLink = /^http.+\/AppNavigation\/Dashboard\/Send\?code=123$/
+    const isReceiveLink = /^get.gdlr.info=123$/
```

Comment:
same comment as above. should match according to custom settings

Reply:
Updated